Makefile

```diff
@@ -269,11 +273,11 @@ $(APIDOCS_ZIP_FILE): $(APIDOCS_HTML_DIR)
 userguide: $(USERGUIDE_HTML_DIR) $(USERGUIDE_ZIP_FILE)
 $(USERGUIDE_HTML_DIR): $(GIT_UG_REPO_PATH) | $(DIST_DIR)
 	$(MAKE) -C doc/users-guide SPHINXBUILD="$(PYTHON) -m sphinx" BUILDDIR=../../build/userguide html
-	-$(RMDIR) $(USERGUIDE_HTML_DIR)
+	-rm -r $(USERGUIDE_HTML_DIR)
```

Comment:
Same here, it looks like the `RMDIR` variable is unused

Reply:
Fixed here: https://github.com/natcap/invest/pull/314/commits/f8a92c70862d88d7dc5889e69dd4afa1141f4d87